runtime/bindings/python/src/pyopenvino/graph/variant.cpp

```diff
@@ -2,16 +2,16 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include "openvino/core/variant.hpp"  // ov::Variant
+#include "pyopenvino/graph/variant.hpp"
 
 #include <pybind11/pybind11.h>
 
-#include "pyopenvino/graph/variant.hpp"
+#include "openvino/core/any.hpp"
 
 namespace py = pybind11;
 
 void regclass_graph_Variant(py::module m) {
-    py::class_<ov::Variant, std::shared_ptr<ov::Variant>> variant_base(m, "Variant");
+    py::class_<ov::Any> variant_base(m, "Variant", py::module_local());
```

Comment:
Why register new class under old name? (same comments as for `PyRTMap`)

Reply:
One of the main requirements of the new common API is that it matches and reflects the C++ API. This particular file breaks this requirement. 
If OV 2.0 introduces a new type `Any` then it should be exposed to Python as `Any`, not as `Variant`.

If `Variant` is eventually removed from the C++ API, it should also not be exposed to Python.

Please adjust the bindings to fulfill this requirement.

Reply:
will be done in separate activity